README.md

```diff
@@ -11,8 +11,10 @@ This repository will hold multiple NodeJS based automation frameworks and exampl
 
 ## Framework examples
 - [WebdriverIO with Jasmine*](./webdriverio)
-    - [WebdriverIO with Webdriver (desktop browser tests)](./webdriverio/web/webdriver)
+    - [WebdriverIO with Webdriver (desktop browser tests)](webdriverio/web/framework)
     - [WebdriverIO with Appium (mobile browser tests)](./webdriverio/web/appium)
+    - [WebdriverIO,cross-platform,parallelization,page objects](./webdriverio/web/framework)
```

Comment:
Not 100% clear why you added this here, now it's double

Reply:
@wswebcreation my customer needs an example of how to use webdriver io, cross-platform, in parallel, with page objects and couldn't find it in our repo

Reply:
Then maybe add it to the current link instead of creating a double one that redirects to the same folder.
Can you also remove your current folder structure and keep the current one? Just a check-check-double-check, did you first merge with master before you submitted this PR?

Reply:
@wswebcreation I don't understand your request for this "Then maybe add it to the current link instead of creating a double one that redirects to the same folder"? With other repos this is what we do because customers come there for use cases. So it's okay to point multiple use cases at a single code base. A very typical example is _parallelization with a framework, cross-browser with a framework, page objects with a framework_, those 3 use cases typically point to the same exact code base.
I'm going to resolve conflicts now